core/src/main/scripts/generate-thrift.sh

```diff
@@ -78,6 +78,9 @@ done
 find $BUILD_DIR/gen-java -name '*.java' -exec grep -Zl '^public class ' {} + | xargs -0 sed -i -e 's/^[}]$/  private static void unusedMethod() {}\
 }/'
 
+# Remove usage of the javax.annotation.Generated because its no longer present in Java 11
+find $BUILD_DIR/gen-java -name "*.java" | xargs sed -i -e '/[@]javax[.]annotation[.]Generated.*/d'
```

Comment:
There's actually a thrift generator flag to omit these.

Instead of: `java:generated_annotations=undated`, we should use `java:generated_annotations=suppress`

Reply:
We could also do this for the 1.9 branch.